Firestore/core/src/firebase/firestore/local/leveldb_transaction.cc

```diff
@@ -203,8 +203,9 @@ void LevelDbTransaction::Commit() {
   }
 
   Status status = db_->Write(write_options_, &batch);
-  FIREBASE_ASSERT_MESSAGE(status.ok(), "Failed to commit transaction: %s",
-                          status.ToString().c_str());
+  FIREBASE_ASSERT_MESSAGE(status.ok(),
```

Comment:
So the old implementation would log what it was writing before it wrote it. This is now missing in this implementation:
```objc
NSString *description = [group description];
FSTLog(@"Committing %@", description);	

Status status = [group writeToDB:_ptr];	
if (!status.ok()) {	
  FSTFail(@"%@ failed with status: %s, description: %@", group.action, status.ToString().c_str(),	
          description);	
}
```